package.json

```diff
@@ -30,6 +30,7 @@
   "scripts": {
     "dev": "yarn react-scripts start",
     "build": "env-cmd -f ./.env react-scripts build",
+    "build:github": "node scripts/setup.js && react-scripts build",
```

Comment:
Why do we need this?

Reply:
Yes this is part of the command run by the electron actions. This is a separate script to avoid using env files.

Reply:
OK.